### PR TITLE
fix(rbd): tuto errors

### PR DESCRIPTION
**Pull request by ldecarvalho-doc**

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

File: managed-databases/postgresql-and-mysql/api-cli/import-data-to-managed-postgresql-databases.mdx [hunk 1]
```diff
@@ -83,7 +83,7 @@ To complete the following procedure, you must have:
 
 - Installed [pg_Admin](https://www.pgadmin.org/).
 
-- Ensured your PGAdmin version is compatible with the PostgreSQL version of the database you will import your data into. Currently, pgAdmin 7.8 or newer is required to fully work with PostgreSQL 16.
+- Ensured your PGAdmin version is compatible with the PostgreSQL version of the database you will import your data into. Refer to the official [pgadmin documentation](https://www.pgadmin.org/docs/pgadmin4/7.8/release_notes.html) to check the compatibility.
```
Comment (jcirinosclwy) [inline managed-databases/postgresql-and-mysql/api-cli/import-data-to-managed-postgresql-databases.mdx:86]:
```suggestion
- Ensured your pgAdmin version is compatible with the PostgreSQL version of the database you will import your data into. Refer to the official [pgAdmin documentation](https://www.pgadmin.org/docs/pgadmin4/7.8/release_notes.html) to check the compatibility.
```

File: managed-databases/postgresql-and-mysql/api-cli/import-data-to-managed-postgresql-databases.mdx [hunk 2]
```diff
@@ -83,7 +83,7 @@ To complete the following procedure, you must have:
 
 - Installed [pg_Admin](https://www.pgadmin.org/).
```
Comment (jcirinosclwy) [inline managed-databases/postgresql-and-mysql/api-cli/import-data-to-managed-postgresql-databases.mdx:84]:
```suggestion
- Installed [pgAdmin](https://www.pgadmin.org/).
```

**Review by jcirinosclwy — APPROVED** [review]

File: managed-databases/postgresql-and-mysql/api-cli/import-data-to-managed-postgresql-databases.mdx [hunk 3]
```diff
@@ -153,7 +153,7 @@ To avoid any statement timeout errors with Managed Database for PostgreSQL (e.g.
     - **Password**
 
     <Message type="note">
-    You can find this information on the [overview page](https://console.scaleway.com/rdb/instances) of your database. The password is the [IAM secret key](/identity-and-access-management/iam/concepts/#api-key) corresponding to the [IAM user](/identity-and-access-management/iam/concepts/#user) or [IAM application](/identity-and-access-management/iam/concepts/#application) you are connecting with. 
+    You can find this information on the [overview page](https://console.scaleway.com/rdb/instances) of your database. The password is the [IAM secret key](/identity-and-access-management/iam/concepts/#api-key) corresponding to the [IAM user](/identity-and-access-management/iam/concepts/#user) or [IAM application](/identity-and-access-management/iam/concepts/#application) you are connecting with.
```
Comment (scw-wtimmermans) [inline managed-databases/postgresql-and-mysql/api-cli/import-data-to-managed-postgresql-databases.mdx:156]: I forgot this one sorry

We also need to replace it with this: "You can find this information on the [overview page](https://console.scaleway.com/rdb/instances) of your database. These are the username and password that you defined upon Database Instance creation."

File: managed-databases/postgresql-and-mysql/api-cli/import-data-to-managed-postgresql-databases.mdx [hunk 4]
```diff
@@ -41,7 +41,7 @@ The import procedure depends on your data source:
 
 #### Prerequisites
 
-To complete this procedure, you must have installed PostgreSQL 16 (or newer) with [pg_dump](https://www.postgresql.org/docs/current/app-pgdump.html) and [pg_restore](https://www.postgresql.org/docs/current/app-pgrestore.html) (bundled with the default PostgreSQL installation).
+To complete this procedure, you must have installed PostgreSQL 15 (or older) with [pg_dump](https://www.postgresql.org/docs/current/app-pgdump.html) and [pg_restore](https://www.postgresql.org/docs/current/app-pgrestore.html) (bundled with the default PostgreSQL installation).
```
Comment (scw-wtimmermans) [inline managed-databases/postgresql-and-mysql/api-cli/import-data-to-managed-postgresql-databases.mdx:44]: To complete this procedure, you must have installed PostgreSQL in a version compatible with your target Managed PostgreSQL, with [pg_dump](https://www.postgresql.org/docs/current/app-pgdump.html) and [pg_restore](https://www.postgresql.org/docs/current/app-pgrestore.html) (bundled with the default PostgreSQL installation).

File: managed-databases/postgresql-and-mysql/api-cli/import-data-to-managed-postgresql-databases.mdx [hunk 5]
```diff
@@ -417,7 +417,7 @@ To complete the following procedure, you must have:
 
 To complete the following procedure, you must have:
 
-- Installed PostgreSQL 16 (or newer) with [pg_restore](https://www.postgresql.org/docs/current/app-pgrestore.html) (bundled with the default PostgreSQL installation).
+- Installed PostgreSQL 15 (or older) with [pg_restore](https://www.postgresql.org/docs/current/app-pgrestore.html) (bundled with the default PostgreSQL installation).
```
Comment (scw-wtimmermans) [inline managed-databases/postgresql-and-mysql/api-cli/import-data-to-managed-postgresql-databases.mdx:420]: Installed PostgreSQL in a version compatible with your target Managed PostgreSQL, with [pg_restore](https://www.postgresql.org/docs/current/app-pgrestore.html) (bundled with the default PostgreSQL installation).

File: managed-databases/postgresql-and-mysql/api-cli/import-data-to-managed-postgresql-databases.mdx (hunk 4 above)
Comment (ldecarvalho-doc) [reply managed-databases/postgresql-and-mysql/api-cli/import-data-to-managed-postgresql-databases.mdx:44]:
```suggestion
To complete this procedure, you must have installed PostgreSQL in a version compatible with your target Managed PostgreSQL, with [pg_dump](https://www.postgresql.org/docs/current/app-pgdump.html) and [pg_restore](https://www.postgresql.org/docs/current/app-pgrestore.html) (bundled with the default PostgreSQL installation).
```

File: managed-databases/postgresql-and-mysql/api-cli/import-data-to-managed-postgresql-databases.mdx (hunk 5 above)
Comment (ldecarvalho-doc) [reply managed-databases/postgresql-and-mysql/api-cli/import-data-to-managed-postgresql-databases.mdx:420]:
```suggestion
- Installed PostgreSQL in a version compatible with your target Managed PostgreSQL, with [pg_restore](https://www.postgresql.org/docs/current/app-pgrestore.html) (bundled with the default PostgreSQL installation).
```

File: managed-databases/postgresql-and-mysql/api-cli/import-data-to-managed-postgresql-databases.mdx (hunk 3 above)
Comment (ldecarvalho-doc) [reply managed-databases/postgresql-and-mysql/api-cli/import-data-to-managed-postgresql-databases.mdx:156]:
```suggestion
    You can find this information on the [overview page](https://console.scaleway.com/rdb/instances) of your database. These are the username and password that you defined upon Database Instance creation.
```

**Pull request merged**